lib/bootstrap.ts

```diff
@@ -57,4 +60,7 @@ $injector.requireCommand("cloud|lib|version", path.join(__dirname, "commands", "
 
 $injector.requireCommand("account|*list", path.join(__dirname, "commands", "account", "account-list"));
 $injector.requireCommand("account|usage", path.join(__dirname, "commands", "account", "usage"));
+
 $injector.requireCommand("accept|eula", path.join(__dirname, "commands", "accept-eula-command"));
+
+$injector.requireCommand("clean|cloud|workspace", path.join(__dirname, "commands", "clean", "clean-cloud-workspace"));
```

Comment:
this should be `cloud|clean|workspace` - all cloud related commands must start with cloud